lib/graph3d/Graph3d.js

```diff
@@ -327,7 +327,25 @@ Graph3d.prototype.getDistinctValues = function(data, column) {
       distinctValues.push(data[i][column]);
     }
   }
-  return distinctValues;
+  return distinctValues.sort(function(a,b) { return a - b; });
+}
+
+
+Graph3d.prototype.getSmallestDifference = function(data, column) {
```

Comment:
> Also maybe mention in a comment that when you pass an empty data value the function returns undefined.

Also for a single point graph. I went with `null` as return value instead of `undefined`.

Comment added.